frontend/src/components/Staffing/AddEngagementForm.tsx

```diff
@@ -79,25 +93,57 @@ export function AddEngagementForm({
 
   // Handler for radio button group
   function handleRadioChange(event: ChangeEvent<HTMLInputElement>) {
-    setRadioValue(event.target.value);
+    setRadioValue(event.target.value as ProjectState);
   }
 
   // Handler for toggle
   function handleToggleChange() {
     setIsFakturerbar(!isFakturerbar);
   }
 
+  async function submitAddEngagementForm(body: EngagementWriteModel) {
+    const url = `/${organisationName}/bemanning/api/projects`;
+
+    try {
+      const data = await fetch(url, {
+        method: "put",
+        body: JSON.stringify({
+          ...body,
+        }),
+      });
+      return (await data.json()) as ProjectWithCustomerModel;
+    } catch (e) {
+      console.error("Error updating staffing", e);
```

Comment:
Todo: Finne en løsning for å deale med feil

Reply:
Jeg er fan av snackbars. Are u?

Reply:
Har opprettet task for det: #369